.github/workflows/contributors.yml

```diff
@@ -13,4 +13,4 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           round: true
-          svgPath: ../../images/contributors_list.svg
+          svgPath: images/contributors_list.svg
```

Comment:
I already had some doubts about this, maybe this will not work and this action will run with respect to the root of the repository but I think now it is clear so let's merge the change :slightly_smiling_face: .